pkg/m3admin/placement/client.go

```diff
@@ -95,41 +87,30 @@ func (p *placementClient) Delete() error {
 
 // Get will get current placement
 func (p *placementClient) Get() (m3placement.Placement, error) {
-	url := p.url + placementBaseURL
-	resp, err := p.client.DoHTTPRequest(http.MethodGet, url, nil)
+	var (
+		url  = p.url + placementBaseURL
+		resp = &admin.PlacementGetResponse{}
+	)
+	err := p.client.DoHTTPJSONPBRequest(http.MethodGet, url, nil, resp)
 	if err != nil {
 		return nil, err
 	}
-	data := &admin.PlacementGetResponse{}
-	defer func() {
-		ioutil.ReadAll(resp.Body)
-		resp.Body.Close()
-	}()
-
-	um := &jsonpb.Unmarshaler{
-		AllowUnknownFields: true,
-	}
-	if err := um.Unmarshal(resp.Body, data); err != nil {
-		return nil, err
-	}
-	if data.Placement == nil {
+
+	if resp.Placement == nil {
 		return nil, errors.New("nil placement fetch")
 	}
-	p.logger.Debug("placement retreived")
-	return m3placement.NewPlacementFromProto(data.Placement)
+	p.logger.Info("placement retreived")
```

Comment:
Nit: I changed this from Info to Debug recently because this happens on every run of the control loop. Since we refresh every N seconds even with no events this means the operator logs could just be `"placement retrieved"` constantly.

Might be worth keeping this as debug.

Reply:
Ah yeah, this was a bad merge, will revert.